tests/conftest.py

```
@@ -63,7 +64,151 @@ def refresh_databases() -> None:
+    temp_db = output_db_path.with_suffix('.temp.sqlite')
+
+    # Ensure we close all connections properly
+    def close_all_connections(db_path: Path) -> None:
+        """Close all connections to the database"""
+        try:
+            # Try to close any existing connections
+            conn = sqlite3.connect(db_path)
+            conn.execute('PRAGMA optimize')  # Helps ensure cleanup
+            conn.close()
+        except Exception:
+            pass
```

Ruff (0.14.8)

`try`-`except`-`pass` detected, consider logging the exception

(S110)